Data/Text/ICU/BitMask.hs

```diff
@@ -0,0 +1,52 @@
+{-# LANGUAGE DefaultSignatures, ScopedTypeVariables #-}
```

Comment:
This module is pretty awful.

I would like to see the unused functions deleted (probably including the default signature stuff), the coding style made to reflect the rest of the package, and the couple of bits that are still needed simplified so that you're no longer depending on `Control.Monad`.